### PR TITLE
Add new rc.locals.

### PR DESCRIPTION
**Pull request by vext01**
Raising for discussion.

### CODE REVIEW

File: etc/rc.local.linux [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
 #
-# /etc/rc.local
-#
-# This script is executed at the end of each multiuser runlevel.
-# This means that once Krun has rebooted the machine, you will not
-# see a login prompt until all benchmarks have finished.
-#
-# Verify this script by running:
-#    sudo service rc.local start
-#
```
Comment (snim2) [inline etc/rc.local.linux]: I think we need to keep at least some of this comment (e.g. where the file should go, how to verify the script).

File: etc/rc.local.linux [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
+# Edit these variables as needed.
+
+# Where your experiment lives
+EXPERIMENT_DIR=/home/vext01/research/warmup_experiment
```
Comment (snim2) [inline etc/rc.local.linux]: `vext01`?

**Comment by snim2** [comment]
Generally looks OK. I'm not sure if you want to create a fictional user name for the example files?

**Comment by vext01** [comment]
How's this?

**Comment by snim2** [comment]
Looks ready to squash...

**Comment by vext01** [comment]
splat.

**Pull request merged**